src/applications/personalization/profile/components/direct-deposit/DirectDeposit.jsx

```diff
@@ -139,48 +144,91 @@ const DirectDeposit = ({
   return (
     <>
       <Headline>Direct deposit information</Headline>
-      <div id="success" role="alert" aria-atomic="true">
-        <ReactCSSTransitionGroup
-          transitionName="form-expanding-group-inner"
-          transitionAppear
-          transitionAppearTimeout={bankInfoUpdatedAlertSettings.FADE_SPEED}
-          transitionEnterTimeout={bankInfoUpdatedAlertSettings.FADE_SPEED}
-          transitionLeaveTimeout={bankInfoUpdatedAlertSettings.FADE_SPEED}
-        >
-          {!!recentlySavedBankInfo && (
-            <div data-testid="bankInfoUpdateSuccessAlert">
-              <AlertBox
-                status={ALERT_TYPE.SUCCESS}
-                backgroundOnly
-                className="vads-u-margin-top--0 vads-u-margin-bottom--2"
-                scrollOnShow
-              >
-                <SuccessMessage benefit={recentlySavedBankInfo} />
-              </AlertBox>
-            </div>
+      {environment.isProduction() ? (
```

Comment:
users should never even get to this page, it'll be blocked because the DD APIs will fail, but just in case...